### ISSUE TITLE
Campaign Managers can view list of Managed Campaigns

### ISSUE THREAD

**Issue by stevejgordon** [HTBox/allReady#1894]
Breakout from #1802 

Update campaign lister if possible or add a new lister page where a campaign manager can see links to all campaign(s) they have access to manage.

Clicking the link should take them to the admin campaign page.

**Comment by HamidMosalla** [comment]
@stevejgordon Working on it, what campaign manager has access to is determined by the values in `CampaignManager` table, am I right?

**Comment by stevejgordon** [comment]
@HamidMosalla Yes that's correct. That table maps users to campaigns they are allowed to manage.

**Comment by HamidMosalla** [comment]
@stevejgordon I need to decide on how to implement the security aspect of this, what I chose is to create a method in claim extensions something like:
`public static bool IsCampaignManager(this ClaimsPrincipal user)`
Am I going the right path? If so, it mean adding a new `UserType `and new claim, which makes the campaign manager a special user, or I could get the current user's id and check to see if it exist in the `CampaignManager `table, which will show the manage campaign page to every user, I'm not sure.

**Comment by stevejgordon** [comment]
@HamidMosalla We have some authorization concepts now that partially enable this. I've just checked and I have some code I started to update a new UserAuthorizationService that I added a few weeks ago. This service get's hooked up by middleware and will become our single point for user authorization code.

I'll see if I can finish that off this morning and get it checked in, since that will then give you an IsCampaignManager bool on this service.
